app/uk/gov/hmrc/individualsemploymentsapi/controller/EmploymentsController.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsemploymentsapi.controller
+
+import java.util.UUID
+import javax.inject.Singleton
+
+import play.api.hal.Hal._
+import play.api.hal.HalLink
+import play.api.mvc.hal._
+import play.api.mvc.{Action, Controller}
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.individualsemploymentsapi.error.ErrorResponses.MatchNotFoundException
+import uk.gov.hmrc.individualsemploymentsapi.error.Recovery
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.Future.{failed, successful}
+
+class EmploymentsController extends Controller with Recovery {
+
+  def hateoas(matchId: UUID) = Action.async {
```

Comment:
am not convinced by the name of the method "hateos". every endpoint is hateos.
Maybe root(matchId: UUID) is better?